test/source/tests/tests/compose.ts

```diff
@@ -525,6 +526,23 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
       expect(Number(await PageRecipe.getElementPropertyJson(composeBody, 'offsetHeight'))).to.equal(initialHeight);
     }));
 
+    ava.default('compose[global:compose] - saving and rendering a draft with image ', testWithSemaphoredGlobalBrowser('compatibility', async (t, browser) => {
+      let composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
+      await ComposePageRecipe.fillMsg(composePage, { to: 'human@flowcrypt.com' }, 'Test Saving Drafts In Mock');
+      await composePage.page.evaluate(() => {
+        // eslint-disable-next-line max-len
+        $('[data-test=action-insert-image]').val('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAYAAABw4pVUAAAAnElEQVR42u3RAQ0AAAgDIE1u9FvDOahAVzLFGS1ECEKEIEQIQoQgRIgQIQgRghAhCBGCECEIQYgQhAhBiBCECEEIQoQgRAhChCBECEIQIgQhQhAiBCFCEIIQIQgRghAhCBGCEIQIQYgQhAhBiBCEIEQIQoQgRAhChCAEIUIQIgQhQhAiBCEIEYIQIQgRghAhCBEiRAhChCBECEK+W3uw+TnWoJc/AAAAAElFTkSuQmCC')
+          .click();
+      });
+      await ComposePageRecipe.waitWhenDraftIsSaved(composePage);
+      await composePage.close();
+      const appendUrl = 'draftId=draft_testsavingdraftsinmock';
+      composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility', { appendUrl });
+      const body = await composePage.waitAny('@input-body');
+      const image = await body.$eval('img', el => el.getAttribute('src'));
+      expect(image).to.not.be.empty;
```

Comment:
You could actually compare the image src to equal

Reply:
Yes, agree, it's better.
Thanks!